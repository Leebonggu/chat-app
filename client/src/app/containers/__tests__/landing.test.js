import React from 'react';
import { Provider } from 'react-redux';
import thunk from 'redux-thunk';
import configureStore from 'redux-mock-store';
import renderer from 'react-test-renderer';
import Landing from '../landing';

const mockStore = configureStore([thunk]);

test('Landing Container should render given props', () => {
  const initialState = {};
  const store = mockStore(initialState);
  const component = renderer.create(
    <Provider store={store}>
      <Landing />
    </Provider>
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
