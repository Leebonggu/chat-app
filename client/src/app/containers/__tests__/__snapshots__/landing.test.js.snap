// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Container should render given props 1`] = `
<div
  style={
    Object {
      "height": "100rem",
    }
  }
>
  Landing
</div>
`;
